static/app/components/events/interfaces/crashContent/exception/content.tsx

```diff
@@ -38,28 +42,27 @@ type Props = {
   threadId?: number;
 } & Pick<React.ComponentProps<typeof StackTrace>, 'groupingCurrentLevel'>;
 
-type CollapsedExceptionMap = Record<number, boolean>;
+type ExceptionRenderStateMap = Record<number, boolean>;
 
-const useCollapsedExceptions = (values?: ExceptionValue[]) => {
-  const [collapsedExceptions, setCollapsedSections] = useState<CollapsedExceptionMap>(
+const useHiddenExceptions = (values?: ExceptionValue[]) => {
```

Comment:
@isabellaenriquez what do you think of removing this toggle completely (and showing the child exceptions by default)?

Now that you've added auto-collapsing of child exceptions, I think that does what this toggle was meant to do, but better. It would be annoying to need to toggle this and then the exception header to view the stack trace IMO

Reply:
Actually, I'm realizing now that the "hidden" toggle does still do something. Without it, I don't think there is anything to differentiate the levels of hierarchy:

```
- Exception Group 1
    - Child 1
    - Child 2 (exc group)
        - Child 2's child
    - Child 3 
```

In this case Child 2's Child would look like the child of Exception Group 1 since we flatten them 😞